network-api/networkapi/wagtailpages/templates/wagtailpages/tags/primary_page_menu.html

```diff
@@ -1,12 +1,21 @@
 {% load wagtailcore_tags %}
 
 {% if singleton_page == False %}
-<div class="hidden-sm-down d-flex flex-row multipage-nav primary-menu" id="multipage-nav">
-	{% for entry in menu_pages %}
-	<div class="mx-3">
-	  <a class="multipage-link {% if current.title == entry.page.title %}multipage-link-active{% endif %}" href="{% pageurl entry.page %}">{% if entry.depth == 2 %}→ {% endif %}{{ entry.menu_title }}
-	  </a>
-	</div>
-	{% endfor %}
+<div class="container hidden-sm-down my-4">
```

Comment:
tidy up DOM structure here so it aligns with fellowship minisite's (designers like fellowships' nav better)

Reply:
Just a heads up, but some of the alignment on the primary nav may be changing a bit. https://github.com/mozilla/foundation.mozilla.org/pull/1490

Hopefully it's not changing anything that you need to worry about here.

Reply:
cool thanks for the headsup @ScottDowne